src/intro.md

```diff
@@ -4,13 +4,41 @@
 [« Previous](index.html) • [TOC](index.html) • [Next »](processWorkflow.html)
 :::
 
-When the JDK was released under the GPLv2 license in May 2007, the need for a
-detailed document for contributors was anticipated. Translating rather
-sophisticated institutional knowledge into a format suitable for non-Oracle
-employees is always a challenge. This document attempts to address the needs of
-developers who wish to participate in the future of the JDK.
+Welcome to the OpenJDK Developers' Guide!
 
-Comments may be sent to [guide-dev (at) openjdk.java.net](mailto:guide-dev-at-openjdk.java.net).
+The OpejJDK project governs the Java programming language and the Java runtime.
```

Comment:
Not quite.  The Java APIs, JLS, JVMS, and other normative specifications (e.g. Serialization Spec, RMI Spec, etc.) are defined by the Java Community Process (jcp.org).  The OpenJDK Project is the place to collaborate on an open-source implementation of the Java Platform, Standard Edition, and related projects.

Reply:
Right. I'll take that sentence :-)